test/Parser.test.js

```diff
@@ -331,6 +331,39 @@ describe("Parser", () => {
 			"b.Number": "number=123",
 			"b['Number']": "number=123",
 			"b[Number]": "",
+			"'str'.concat()": "string=str",
+			"'str'.concat('one')": "string=strone",
+			"'str'.concat('one').concat('two')": "string=stronetwo",
+			"'str'.concat('one').concat('two', 'three')": "string=stronetwothree",
+			"'str'.concat('one', 'two')": "string=stronetwo",
+			"'str'.concat('one', 'two').concat('three')": "string=stronetwothree",
+			"'str'.concat('one', 'two').concat('three', 'four')": "string=stronetwothreefour",
+			"'str'.concat('one', obj)": "wrapped=['str' string=str]+[null]",
```

Comment:
I guess this could be more efficiently processed, but I guess it would break the code generation because of the assumption that the prefix can be replaced with a string literal.

But as nobody writes such code by hand and babel won't generate this code (first argument to concat is always an expression), we can keep it simple.

Reply:
Yeah when I was originally preprocessing it into a template literal, I was concating `str` and `one` into two quasis, but that didn't end up working because of the quote issue :(